### DISCUSSION TITLE
How can I take user input?

### DISCUSSION THREAD

**Discussion by tsklxiv** [janet-lang/janet#704] [Q&A] [answered]
The question is on the title, how can I take user input in Janet? I found that there is this thing called `stdin` which (as the doc explain) is the standard input file, but I don't know how to use it. I have tried to found information about `stdin` on the Janet Documentation, but also no result.

**Comment by sogaiu** [comment, marked as answer]
Does the following help?
```
(prin "what is your favorite color? ")

(flush)

(def response (getline))

(print "really? your favorite color is " (string/trimr response) "?")
```

**Reply by tsklxiv** [reply]
Yes! It works! Thanks!
